frontend/packages/console-dynamic-plugin-sdk/src/extensions/index.ts

```diff
@@ -0,0 +1,12 @@
+export * from './add-actions';
```

Comment:
I re-added the index file to have an entry point for `webpack` build and single file .d.ts which is generated by `DtsBundleWebpack`

Reply:
we need 3 different entries
 - 1. webpack (ConsoleRemotePlugin) so dynamic plugin wont bundle webpack in the build
 - 2. api - so we can replace the import with webpack externals
 - 3. extensions/index - the rest